Makefile

```diff
@@ -93,6 +93,16 @@ safety: ## Runs `safety check` to check python dependencies for vulnerabilities
 		|| exit 1; \
 	done
 
+.PHONY: update-pip-requirements
+update-pip-requirements: ## Updates all Python requirements files via pip-compile.
+	pip-compile --generate-hashes --output-file securedrop/requirements/admin-requirements.txt \
+		securedrop/requirements/ansible.in
+	pip-compile --output-file securedrop/requirements/develop-requirements.txt \
+		securedrop/requirements/ansible.in \
+		securedrop/requirements/develop-requirements.in
+	pip-compile --output-file securedrop/requirements/test-requirements.txt \
+		securedrop/requirements/test-requirements.in
```

Comment:
Great catch on the missing line in the Makefile target! Added via 8207ab6dae39f5bbdcaaf0301cc920268c4c528e. However, I have **not updated the securedrop-requirements.txt** file. Doing so will bump several versions and require changes to the config tests (since we hardcode version strings for a lot of pip libs, since the update story used to be quite rickety). 

Also agree with your suggestion for a rename: `securedrop-app-code-requirements` seems most explicit to me, since that's the name of the package in which they're shipped. @redshiftzero could you open a follow up issue to tackle actually updating those dependencies after merge of this PR?

Reply:
done, followup in #2554